staging/vhost-device-gpu/src/vhu_gpu.rs

```diff
@@ -299,10 +279,10 @@ impl VhostUserGpuBackend {
 
                 virtio_gpu.transfer_read(ctx_id, resource_id, transfer, None)
             }
-            GpuCommand::CmdSubmit3d(info) => {
+            GpuCommand::CmdSubmit3d(_info) => {
                 todo!()
             }
-            GpuCommand::ResourceCreateBlob(info) => {
+            GpuCommand::ResourceCreateBlob(_info) => {
```

Comment:
we can start practicing how to keep unrelated changes/fixups like this in a separate commit in the future. that way we can have the commits tidy.